oam/config.yaml

```diff
@@ -0,0 +1,29 @@
+apiVersion: core.oam.dev/v1beta1
+kind: Application
+metadata:
+  name: config-example
+  annotations:
+    version: v0.0.1
+    description: "This is my app"
+spec:
+  components:
+    - name: webcap
+      type: component
+      properties:
+        image: wasmcloud.azurecr.io/http-hello-world:0.1.0
+        # You can pass any config data you'd like sent to your component as a string->string map
+        config:
+          - name: raw_config
+            properties:
+              lang: EN-US
+
+    - name: webcap
+      type: capability
+      properties:
+        image: wasmcloud.azurecr.io/httpserver:0.17.0
+        # You can pass any config data you'd like sent to your provider as a string->string map
+        config:
+          - name: raw_provider_config
+            properties:
+              raw: json
+              data: "{}"
```

Comment:
```suggestion
              data: |
              {}
```

Reply:
Spacing might not be right here, but probably going to be easier tor read/fill in for people that way

Reply:
Ah so here I specifically meant to give it a string, let me make that more clear

Reply:
Ah sorry this wasn't clear -- the [YAML `|` operator ("literal scalar style)](https://yaml-multiline.info/) is *for* strings -- what it helps you do is write the string in a more natural way without having to escape stuff. 

Like someone trying to put data in there is immediately going to have to try and escape `"` while writing JSON, which is really annoying